### PR TITLE
Select, Delete operations using BoardState and ClientBoardState

### PR DESCRIPTION
**Pull request by wildcat26**
Fixes convict-git/KoraKaagaz#80

### CODE REVIEW

File: src/processing/SelectDelete.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
+package processing;
+
+import java.util.*;
+import processing.utility.*;
+import processing.boardobject.*;
+
+
```
Comment (satchit24) [inline src/processing/SelectDelete.java:13]: Please add author's name and reviewer's name.

File: src/processing/SelectDelete.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        BoardObject selected = board.getBoardObjectFromId(mostProbableObjectId);
+        if (selected == null) {
+            board.setSelectedObject(new PriorityQueueObject(null, null));
+            return new ArrayList<Position>();              // returning empty list
+        }
+
+        board.setSelectedObject(new PriorityQueueObject(selected.getObjectId(), selected.getTimestamp()));
+        return selected.getPositions();
+    }
+
```
Comment (satchit24) [inline src/processing/SelectDelete.java:61]: Author can you comment select section as done in delete section with comments in multi-lines.
Other than that looks good :-)

Comment (wildcat26) [reply src/processing/SelectDelete.java:61]: the `TODO` line needs one function call too

**Comment by wildcat26** [comment]
@satchit24 @sakshi1603  Open for review

**Comment by wildcat26** [comment]
@satchit24 @sakshi1603 `giveSelectedPixels` will be implemented by UI. Thus, its path is unknown yet.

File: src/processing/SelectDelete.java (hunk 2 above)
Comment (wildcat26) [reply src/processing/SelectDelete.java:61]: done

**Review by satchit24 — APPROVED** [review]

**Pull request merged**